src/models/metadata.js

```diff
@@ -2,7 +2,6 @@
 
 function getMetadata (config) {
   return {
-    public_key: config.getIn(['keys', 'ed25519', 'public']),
```

Comment:
It's not configured with ed25519 keys, which are used by the ledger and notary for signing crypto conditions. I think this must have been a copy/paste error